sql/pokemon-raid-bot.sql

```diff
@@ -25,6 +25,7 @@ CREATE TABLE `attendance` (
   `late` tinyint(1) unsigned DEFAULT '0',
   `invite` tinyint(1) unsigned DEFAULT '0',
   `pokemon` varchar(20) DEFAULT '0',
+  `alarm` int(11) unsigned NULL DEFAULT '0',
```

Comment:
Am I missing something? Why int(11)? Either boolean or tinyint(1) is enough, or why do you always do „alarm = alarm+1“ and MOD to check if it’s even/uneven? 🤷‍♂️🤔

Reply:
```
`alarm` tinyint(1) unsigned DEFAULT '0',
```